api/pom.xml

```diff
@@ -17,183 +17,28 @@
-                        <docfilessubdirs>true</docfilessubdirs>
-                        <attach>true</attach>
-                        <doclint>none</doclint>
-                        <doctitle>${project.name}</doctitle>
-                        <windowtitle>${project.name}</windowtitle>
-                        <header><![CDATA[<br>${project.name} v${project.version}]]></header>
-                        <bottom>
-<![CDATA[
-        Copyright &#169; ${inceptionYear} &#x2013; {currentYear} {organizationName}. All rights reserved.
-        <br>
-        Use is subject to <a href="{@docRoot}/doc-files/license.html" target="_top">license terms</a>.
-    ]]>
-                        </bottom>
-                        <tags>
-                            <tag>
-                                <name>implSpec</name>
-                                <placement>a</placement>
-                                <head>Implementation Specification:</head>
-                            </tag>
-                        </tags>
-                    </configuration>
-                <executions>
-                    <execution>
-                        <id>attach-javadocs</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-source-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>attach-sources</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>biz.aQute.bnd</groupId>
-                <artifactId>bnd-maven-plugin</artifactId>
-                <version>${bnd.version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>bnd-process</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <configuration>
-                    <archive>
-                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-                    </archive>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>biz.aQute.bnd</groupId>
-                <artifactId>bnd-baseline-maven-plugin</artifactId>
-                <version>${bnd.version}</version>
-                <configuration>
-                    <base>
-                        <version>1.4</version>
```

Comment:
How did you specify the baseline version?

Reply:
The version defaults to the highest version of the project's artifact that is less than the version of the project's artifact. Meaning that with this change it will be `2.0`. If we want to use another version, we can just override the plugin definition and configuration coming from Parent.

Reply:
I tried to use the latest version from maven central before. However, once you deliver RCx to maven central, the RCx will be used. Between the RCm, RCn, we are not allowed to update mistakes wihout version updates. I raised an [issue](https://github.com/bndtools/bnd/issues/4240) with bnd to ask for a facility on just pulling in released versions. The issue was solved in the bnd plugin 5.2. Let me create a PR on parent.pom to pull in the latest version and specify a property and then this can be done. [missed the bnd release notes, hence my previous question.]

Reply:
fixed the parent pom.xml via this [PR](https://github.com/eclipse/microprofile-parent/pull/37)

Reply:
Thanks!